src/components/RoundScore/RoundScore.js

```diff
@@ -0,0 +1,148 @@
+import { h } from 'preact';
+import { formatScoreTime } from 'common/formatters';
+import cn from 'classnames';
+import Icon from 'react-fontawesome';
+import { holyWhite, winningGreen, superLightGreen, secondaryActionColor, partialCheater } from 'styles/colour-palette'
+
+const Solution = {
+    CORRECT: 'correct',
+    PARTIAL: 'partial'
+};
+
+function generateEntries(score) {
+    return score.map((entry) => {
+        const rootClasses = {
+            'score-entry': true,
+            '-solved': entry.correct === Solution.CORRECT,
+            '-partial': entry.correct === Solution.PARTIAL,
+        };
+        return (
+            <div key={entry.participantId} className={cn(rootClasses)}>
+                <div className="-name">{entry.displayName}</div>
+                <div className="-time">{formatScoreTime(entry.time)}</div>
+                <div className="-selector">{entry.code}</div>
+                <style jsx>{`
+                .score-entry {
+                    display: flex;
+                    line-height: 3em;
+                    color: ${holyWhite};
+                    border-bottom: 1px solid ${superLightGreen};
+                }
+                .score-entry:hover {
+                    background: ${secondaryActionColor};
+                    cursor: pointer;
+                }
+                `}</style>
+            </div>
+        );
+    });
+}
+
+function getSolvedCount(score) {
+    return score.filter(s => s.correct === Solution.CORRECT).length;
+}
+
+const RoundScore = ({ score }) => (
+    <div className="round-score">
+        <style jsx>{`
+            .round-score {
+                display: flex;
+                flex-direction: column;
+            }
+        `}</style>
+        <div className="header">
+            <div className="stats">
+                <style jsx>{`
+                    .header .stats {
```

Comment:
no need to nest selectors, `styled-jsx` resolves where `<style>` tag is placed and generates unique classnames

Reply:
The idea here was to keep related selectors close to nodes. Also, for some reason, `styled-jsx`doesn't generate unique classes, all were global. Anyway, I agree that there is no necessity and I can refactor.

Reply:
> for some reason, styled-jsxdoesn't generate unique classes

that is weird... 🤔  Will investigate this
We can leave nested selectors for now

Reply:
Fixed with changing the order of babel plugins. It doesn't work after preact